ghutil/mock_ghutil.go

```diff
@@ -19,8 +19,9 @@
 package ghutil
 
 import (
+	context "context"
```

Comment:
I think you cleaned up other imports; should this just be `context`?

Reply:
MockGen generates this file (I re-generated it using `make mock`), I can manually clean up the imports here, but it will cause the `make test` on this folder to fail. 
Same for the others.

Reply:
My apologies; you're absolutely right, I missed that this was a generated file.